CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the Wazuh app project will be documented in this file.
 
+## Wazuh v4.1.3 - Kibana 7.10.0 , 7.10.2 - Revision 4106
```

Comment:
move to `v4.2.0` and `Revision 4201` please

Reply:
Done!